webservices/resources/candidates.py

```diff
@@ -194,14 +176,12 @@ def get(self, **kwargs):
 
         return data
 
-    def get_candidate(self, args, page_num, per_page, candidate_id, committee_id):
```

Comment:
We need to put back in support for passing in `committee_id` since this still powers `/committee/<id>/candidates`